guides/doc-Administering_Red_Hat_Satellite/topics/Monitoring_Capsules.adoc

```diff
@@ -28,7 +28,7 @@ In addition, you can answer to the following questions:
 *Which hosts are managed by the {SmartProxyServer}?*:: The number of associated hosts is displayed next to the *Hosts managed* label.
 Click the number to view the details of associated hosts.
 
-*How much storage space is available on the {SmartProxyServer}?*:: The amount of storage space occupied by the Pulp content in `/var/lib/pulp`, `/var/lib/pulp/content`, and `/var/lib/mongodb` is displayed.
+*How much storage space is available on the {SmartProxyServer}?*:: The amount of storage space occupied by the Pulp content in `/var/lib/pulp` is displayed.
```

Comment:
Is the disk space check implemented? At least the Smart Proxy plugin no longer implements it. Is is available via the Pulp API, but do verify this is still present in the UI.

Reply:
Its currently not working correctly, but thats a bug: https://projects.theforeman.org/issues/31506

we're hoping to get it in for 4.0, so i think assuming its there is fine.